crates/core_arch/src/wasm32/simd128.rs

```diff
@@ -297,80 +297,80 @@ pub unsafe fn v128_load(m: *const v128) -> v128 {
 #[inline]
 #[cfg_attr(test, assert_instr(v128.load8x8_s))]
 #[target_feature(enable = "simd128")]
-pub unsafe fn v128_load8x8_s(m: *const i8) -> v128 {
-    transmute(simd_cast::<_, i16x8>(*(m as *const i8x8)))
+pub unsafe fn i16x8_load_extend_i8x8(m: *const i8) -> v128 {
+    transmute(simd_cast::<_, simd::i16x8>(*(m as *const simd::i8x8)))
 }
 
 /// Load eight 8-bit integers and zero extend each one to a 16-bit lane
 #[inline]
 #[cfg_attr(test, assert_instr(v128.load8x8_u))]
 #[target_feature(enable = "simd128")]
-pub unsafe fn v128_load8x8_u(m: *const u8) -> v128 {
-    transmute(simd_cast::<_, u16x8>(*(m as *const u8x8)))
+pub unsafe fn u16x8_load_extend_u8x8(m: *const u8) -> v128 {
```

Comment:
```suggestion
pub unsafe fn i16x8_load_extend_u8x8(m: *const u8) -> v128 {
```
This makes the naming consistent with `i16x8_extend_low_u8x16`. Same with the other 3 below.

Reply:
Makes sense to me, done now!